modules/API/pages/gsql-endpoints.adoc

```diff
@@ -6447,6 +6460,38 @@ Sample Response::
 
 '''
 
+[#_delete_udt_files]
+=== Remove user-defined function files
```

Comment:
@zhihuapengtg Please review this section.

Note that you had not provided us with an example, so I created one. I guessed at the response, based on looking at similar endpoints.